pkg/runtime/application_config_provider.go

```diff
@@ -21,7 +23,13 @@ const externalEvents = "externalEvents"
 var databaseConfig = config.MustRegisterSection(database, &interfaces.DbConfigSection{})
 var flyteAdminConfig = config.MustRegisterSection(flyteAdmin, &interfaces.ApplicationConfig{})
 var schedulerConfig = config.MustRegisterSection(scheduler, &interfaces.SchedulerConfig{})
```

Comment:
Do you know why are we registering many sections in one package? it works... but not needed I guess....

Reply:
oh just as a logical/organizational breakdown